olive/engine/engine.py

```diff
@@ -910,6 +918,13 @@ def _run_pass(
 
         run_start_time = datetime.now().timestamp()
         try:
+            from olive.passes.onnx.perf_tuning import OrtPerfTuning
+
+            # For perf-tuning, we need the model evaluation happens in target even if it is a pass.
+            # TODO(myguo): consider more better way to handle System doesn't support run pass like DockerSystem
+            if isinstance(p, OrtPerfTuning) and isinstance(host, LocalSystem):
```

Comment:
Is it possible to transfer the `target` system config into `OrtPerfTuning`'s config set? Then we can move the initialization of `target` to specific pass.

Reply:
this kind of logic is transparent to end user. If we choose expose it by default_config, user has to explicitly set the target. this is not the expected behavior.